opus/application/static_media/js/browse.js

```diff
@@ -62,16 +62,17 @@ var o_browse = {
         $(".gallery-contents, .dataTable").on('wheel ps-scroll-up', function(event) {
             let startobs = (opus.prefs.view === "cart" ? "cart_startobs" : "startobs");
             let tab = `#${opus.prefs.view}`;
+            let contentsView = opus.prefs.browse === "gallery" ? ".op-gallery-view" : ".dataTable";
```

Comment:
Thinking on this - seems like you should just create a single unified class, like "op-scroll-container" and then you don't need to do this if so many places.  My first thought was to create a function that returns the class since you are using it multiple places, but i think a unified class is a better way to go.